doc/api.yml

```diff
@@ -1834,6 +1834,7 @@ components:
           type: string
         profile_picture:
           type: string
+          description: Profile picture of User stored in S3
```

Comment:
this is not enough.

describe the format of the profile_picture (PNG ? JPEG?)
Does it contain the folderName ?
Are there restrictions concerning the way you name your file ?
Is it insertable ?